hyper-capi/README.md

```diff
@@ -0,0 +1,13 @@
+# C API for hyper
+
+This provides a C API to use the hyper library.
+
+## Testing
+
+```
+cargo build
+
+gcc -o client examples/client.c -I./include -L./target/debug -lhyper_c
```

Comment:
How exactly is the `hyper_c` lib getting made? I've run `cargo build` in the root dir but I get neither a .so nor a .a file built?